src/components/AdminPortal/Venues/Modal/VenueModal.module.css

```
@@ -0,0 +1,44 @@
+.inputField {
+  margin-top: var(--space-4);
+  margin-bottom: var(--space-4);
+  background-color: var(--eventManagement-button-bg);
```

Stylelint (17.2.0)

[error] Expected custom property name "--eventManagement-button-bg" to be kebab-case (custom-property-pattern)

(custom-property-pattern)